lib/msf/core/exploit/tcp.rb

```diff
@@ -196,11 +196,12 @@ def cleanup
   end
 
   def print_prefix
```

Comment:
I think deleting print_prefix entirely from this file has the same outcome as it just calling super.

Reply:
Yep valid!